flow/Net2.actor.cpp

```diff
@@ -1294,6 +1295,7 @@ Net2::Net2(const TLSConfig& tlsConfig, bool useThreadPool, bool useMetrics)
 	setGlobal(INetwork::enNetworkConnections, (flowGlobalType)network);
 	setGlobal(INetwork::enASIOService, (flowGlobalType)&reactor.ios);
 	setGlobal(INetwork::enBlobCredentialFiles, &blobCredentialFiles);
+	setGlobal(INetwork::enProxy, &proxy);
```

Comment:
is this needed? fdbserver calls `g_network->setGlobal(INetwork::enProxy, opts.proxy);`?

Reply:
g_network maintains a map<ID, address>. This `setGlobal(INetwork::enProxy, &proxy);` sets the address of the net2.proxy to the map. 

In the fdbserver, we should not set the address of opts.proxy as I did, because the opts is a local var.